configs/setups/awiesm/awiesm-2.1.yaml

```
@@ -63,11 +63,12 @@ general:
         - '2.1-wiso'
         - '2.1-recom'
         choose_version:
-          '2.1':
+          '2.1-gamma':
```

```suggestion
          '2.1':
```
Do we really want to get rid of version 2.1? There might be people still wanting to use 2.1 and it has not being deleted from other places. @pgierz , in case you like the idea of leaving `2.1` and adding `2.1-gamma`, this suggestion and the one below should do that.